index.html

```diff
@@ -337,7 +347,7 @@
           scheduleObj.room = "";
         }
         //Add in photos if the user's schedule is being rendered
-        if (type == "user") {
+        if (type == "user" || type == "student") {
```

Comment:
This sort of seems like it should be a parameter on the element (i.e. the 'view'), not on this function, which constructs the 'model'.